src/main/test/cz/muni/fi/DaoTests/ComponentDaoTest.java

```diff
@@ -0,0 +1,92 @@
+package cz.muni.fi.DaoTests;
+
+import cz.muni.fi.PersistenceApplicationContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceUnit;
+
+/**
+ * Created by R. Tamas, 445241.
+ */
+
+@ContextConfiguration(classes = PersistenceApplicationContext.class)
+public class ComponentDaoTest extends AbstractTestNGSpringContextTests {
+
+    @PersistenceContext
+    private EntityManager em;
+
+    @PersistenceUnit
+    private EntityManagerFactory emf;
+
+    @Autowired
+    private ComponentDaoImpl componentManager = new ComponentDaoImpl();
+
+
+    private Component engine1 = new Component();
```

Comment:
Pls move these properties to setup() method

Reply:
if i moved them i cannot access them from other methods then just `setup()`

Reply:
sure, checked only engine1 which is not used anywhere else.. i would move engine1 inside and rename the rest